docker/Dockerfile

```diff
@@ -14,11 +14,8 @@ RUN apt-get update -y && \
         clang-3.7 \
         emacs \
         g++ \
-        g++-4.8 \
         gcc \
-        gcc-4.8 \
         gfortran \
-        gfortran-4.8 \
```

Comment:
I think we need a gcc-6.5 in here. We need to have multiple system gccs.

Reply:
Ok, I'll add 6.5.0